live-examples/js-examples/array/meta.json

```diff
@@ -1,190 +1,196 @@
+            "fileName": "array-reduce-right.html",
+            "title": "JavaScript Demo: Array.reduceRight()",
+            "type": "js"
+        },
+        "arrayReduce": {
+            "exampleCode": "./live-examples/js-examples/array/array-reduce.js",
+            "fileName": "array-reduce.html",
+            "title": "JavaScript Demo: Array.reduce()",
+            "type": "js"
+        },
+        "arrayReverse": {
+            "exampleCode": "./live-examples/js-examples/array/array-reverse.js",
+            "fileName": "array-reverse.html",
+            "title": "JavaScript Demo: Array.reverse()",
+            "type": "js"
+        },
+        "arrayShift": {
+            "exampleCode": "./live-examples/js-examples/array/array-shift.js",
+            "fileName": "array-shift.html",
+            "title": "JavaScript Demo: Array.shift()",
+            "type": "js"
+        },
+        "arraySlice": {
+            "exampleCode": "./live-examples/js-examples/array/array-slice.js",
+            "fileName": "array-slice.html",
+            "title": "JavaScript Demo: Array.slice()",
+            "type": "js"
+        },
+        "arraySome": {
+            "exampleCode": "./live-examples/js-examples/array/array-some.js",
+            "fileName": "array-some.html",
+            "title": "JavaScript Demo: Array.some()",
+            "type": "js"
+        },
+        "arraySort": {
+            "exampleCode": "./live-examples/js-examples/array/array-sort.js",
+            "fileName": "array-sort.html",
+            "title": "JavaScript Demo: Array.sort()",
+            "type": "js"
+        },
+        "arraySplice": {
+            "exampleCode": "./live-examples/js-examples/array/array-splice.js",
+            "fileName": "array-splice.html",
+            "title": "JavaScript Demo: Array.splice()",
+            "type": "js"
+        },
+        "arrayToLocaleString": {
+            "exampleCode": "./live-examples/js-examples/array/array-tolocalestring.js",
+            "fileName": "array-tolocalestring.html",
+            "title": "JavaScript Demo: Array.toLocaleString()",
+            "type": "js"
+        },
+        "arrayToString": {
+            "exampleCode": "./live-examples/js-examples/array/array-tostring.js",
+            "fileName": "array-tostring.html",
+            "title": "JavaScript Demo: Array.toString()",
+            "type": "js"
+        },
+        "arrayUnshift": {
+            "exampleCode": "./live-examples/js-examples/array/array-unshift.js",
+            "fileName": "array-unshift.html",
+            "title": "JavaScript Demo: Array.unshift()",
+            "type": "js"
+        },
+        "arrayValues": {
+            "exampleCode": "./live-examples/js-examples/array/array-values.js",
+            "fileName": "array-values.html",
+            "title": "JavaScript Demo: Array.values()",
+            "type": "js"
+        },
+        "arrayIterator": {
```

Comment:
This should be given in alphabetical order. Also I'm not sure why this diff includes the entire file, do you know?